clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -100,7 +90,8 @@ class AzureResourceManagerClient extends AzureBaseClient {
     ResourceGroupListParameters parameters = new ResourceGroupListParameters()
     parameters.setTagName("filter")
     parameters.setTagValue(applicationName)
-    return this.getResourceManagementClient(creds).getResourceGroupsOperations().list(parameters).resourceGroups
+
+    this.getResourceManagementClient(creds).getResourceGroupsOperations().list(parameters).resourceGroups
   }
 
   boolean resourceGroupExists(AzureCredentials creds, String resourceGroupName) {
```

Comment:
This method doesn't make sense to me.  The resourceGroupName isn't used and the return value of the method probably isn't a boolean.